receiver/gitproviderreceiver/metadata.yaml

```diff
@@ -42,21 +42,56 @@ metrics:
   git.repository.count:
     enabled: true
     description: Number of repositories in an organization
-    unit: "{repository}"
+    unit: '{repository}'
     gauge:
       value_type: int
     attributes: []
   git.repository.branch.count:
     enabled: true
     description: Number of branches in a repository
-    unit: "{branch}"
+    unit: '{branch}'
     gauge:
       value_type: int
     attributes: [repository.name]
+  git.repository.branch.time:
+    enabled: true
+    description: Time the branch has existed.
+    unit: s
+    gauge:
+      value_type: int
+    attributes: [repository.name, branch.name]
+  git.repository.branch.commit.aheadby_count:
```

Comment:
I'm wondering if the metric name here should be `aheadby.count`. I'm not super well educated on semantic conventions for naming, but it seems like those options would fit better than the [current naming schema ruels](https://opentelemetry.io/docs/specs/semconv/general/metrics/#use-count-instead-of-pluralization-for-updowncounters).

(Same comment for the rest of the `_count` metric names)

Reply:
Yup, agreed, I'll make that adjustment. That's actually what we already have in the source, I made the change locally on this PR because of a failed memory from another conversation. Thanks for sending the reference doc.